### ISSUE TITLE
Generator train_data is same with test_data ,only shuffle is different

### ISSUE THREAD

**Issue by MichaelGou1105** [WangLabTHU/deepseed#3]
code 

https://github.com/WangLabTHU/deepseed/blob/af91290df1e3839ad6b1d02c5c61e7378b2622e3/Generator/cGAN_training.py#L193

```
def main():
    data_name = 'ecoli_mpra_3_laco'
    seqL = 165
    train_data, test_data = DataLoader(LoadData(is_train=True, path='../data/{}.csv'.format(data_name), split_r=1),
                                       batch_size=32, shuffle=True), DataLoader(LoadData(is_train=True, path='../data/{}.csv'.format(data_name), split_r=1), batch_size=32)
```

**Comment by WangLabTHU** [comment]
Thank you. The changes have been successfully merged into the main branch!
